llvmlite/tests/test_ir.py

```diff
@@ -145,6 +145,22 @@ def test_declare_attributes(self):
         # Check pickling
         self.assert_pickle_correctly(func)
 
+    def test_declare_convergent_attribute(self):
+        # Now with function attributes
+        func = self.function()
+        func.attributes.add("convergent")
```

Comment:
To save repeating test code, could this line perhaps just be added into the above test? I think all that needs checking is that `convergent` wires through as an attribute, regardless of whether it makes sense for a `__gxx_personality` ?